packages/roosterjs-editor-dom/lib/utils/setColor.ts

```diff
@@ -23,7 +38,6 @@ export default function setColor(
                 ? modeIndependentColor?.darkModeColor
                 : modeIndependentColor?.lightModeColor) || colorString
         );
-
         if (element.dataset) {
```

Comment:
nit, do not remove the empty line before "if", normally we should add empty line before those structural statement such as if, for, while, ...